releasenotes/notes/fix-rcm-soft-forksafe-7e6d6b59495e097c.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Ensure that worker threads that run on start-up are recreated at the right
```

Comment:
I think it's worth noting that this only impacts Python versions < 3.7 (`_soft` should have no effect after `os.register_at_fork`) is available